combat-trainer.lic

```diff
@@ -1627,6 +1628,22 @@ class SpellProcess
     @use_auto_mana = data['use_auto_mana']
     game_state.action_taken if (@skill == 'Targeted Magic' || @prep == 'target') && game_state.weapon_skill == 'Targeted Magic'
   end
+  
+  def check_face(game_state)
+    return unless Flags['ct-face-what'] || DRRoom.dead_npcs.size > 0
+    if (game_state.npcs.length > 0)
+        case bput("eng #{game_state.npcs.first}", 'You will have to retreat from your current melee first', 'You are already at melee', 'You begin to advance', 'You spin around to face', 'is already quite dead')
```

Comment:
The internals of this if statement are tabbed instead of 2 spaced for alignment.

Reply:
fixed